### ISSUE TITLE
Uploading replacement video doesn't work

### ISSUE THREAD

**Issue by ocrasborn** [Woseseltops/signbank#66]
For some reason, we can no longer upload a replacement video (just tried for LEEUW-B): the video appears with suffix -1 in the folder glossvideo/LE/, but the interface keeps showing the old video.

**Comment by Woseseltops** [comment]
I cannot reproduce this; after trying for some time with my own testgloss and a testvideo, I tried moving your LEEUW_B_1 video to the import_gloss folder, and then visited signbank.science.ru.nl/dictionary/import_videos/ , and as far as I can see it now has the new video there (but please check!). http://signbank.science.ru.nl/dictionary/gloss/22/

Any idea what I can do to reproduce this problem? Or maybe it was a typo in the name from your side?

**Comment by ocrasborn** [comment]
[reminder: test videos LEEUW/LEEUWARDEN, led to error for Onno]

**Comment by Woseseltops** [comment]
Related to #65

**Comment by Woseseltops** [comment]
Fixed: it turns out this wasn't about the batch upload functionality, as I assumed, but about the web interface. This functionality had a problem with finding the correct path of the original video (in particular when that video was added in batch). I've now fixed that, and added that the original videos will be renamed original_name_1, original_name_2, etc. 

I've also tried deleting videos, and that works too (as of this fix?).

PS
This solution still kind of ignores the whole video system, as can be accessed in the admin. I saw no reason to use it yet.
